src/components/axes/timeAxis.ts

```diff
@@ -202,9 +205,10 @@ export module Axis {
     public axisConfigurations(configurations: TimeAxisConfiguration[]): Time;
     public axisConfigurations(configurations?: any): any {
       if(configurations == null){
-        return this._possibleTimeAxisConfigurations;
+        return this._currentTimeAxisConfigurations;
       }
-      this._possibleTimeAxisConfigurations = configurations;
+      this._currentTimeAxisConfigurations = configurations;
+      this._maximumTiers = _Util.Methods.max(configurations.map((config: TimeAxisConfiguration) => config.length), 0);
```

Comment:
it depends how often the `_computeHeight()` is called. I will double check tomorrow. I personally prefer it not to be an instance variable, but it might be an overhead to calculate the max number of Tiers every time we compute the height